src/components/Modal/index.tsx

```diff
@@ -28,12 +28,24 @@ export interface ModalProps extends StandardProps {
    * Event triggered when the modal should be closed by the user.
    */
   onClose?(e: ModalCloseEvent): void;
+  /**
+   * Show or hide close modal button
+   * @default true
+   */
+  showCloseButton?: boolean;
```

Comment:
Not sure if this is needed, e.g., if `onClose` is unset then the button should do nothing --> do not display the button then. Otherwise, I would always tend *not to* introduce *by default* `true` booleans -> they should be `false` such that they are implicit. As such I would (personally) call such a prop (if there really should be one) `hideCloseButton`.

Reply:
removed